_candidates_en/kmizu_1.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Kota Mizushima (kmizu)
+title: "Gentle introduction to DOT calculus"
+length: 40
+audience: Advanced
+language: Japanese
+twitter: kmizu
+github: kmizu
+icon: https://pbs.twimg.com/profile_images/703094910417678337/b_v8jTJ3_400x400.jpg
+organization: Dwango, Inc
+tags:
+  - Functional Programming, Others
+suggestions:
+  - Those interested in programming language theory
+  - Those interested in the theoretical foundation of Scala
+  - Those who want to understand Scala deeoper
```

Comment:
Line 16 Original:  - Those who want to understand Scala deeoper
Change: - Those who want to understand Scala deeper